README.md

```
@@ -1,5 +1,981 @@
+        - 뉴스 기사에 댓글을 등록할 수 있습니다.
+        - 자신의 댓글을 수정하고 삭제할 수 있습니다.
+        - 특정 뉴스 기사에 대한 댓글 목록을 `날짜`와 `좋아요 수`로 정렬하여 조회할 수 있습니다.
+        - 좋아요를 등록하고 취소할 수 있습니다.
+    - 활동 내역 관리
+        - 사용자 별로 사용자의 정보와 구독 중인 관심사, 최근 작성한 댓글, 좋아요를 누른 댓글, 최근 확인한 뉴스 기사를 조회할 수 있습니다.
+    - 알림 관리
+        - 구독 중인 관심사에 대한 뉴스기사가 등록되거나 자신의 댓글이 좋아요를 받으면 알림을 받을 수 있습니다.
+        - 확인하지 않은 알림 목록을 최신순으로 조회할 수 있습니다.
+        - 알림을 개별 확인 또는 전체 확인 할 수 있고, 일주일 후에 확인한 알림을 자동으로 삭제합니다.
+- 주요 화면
+  |Tab|화면|||
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)